src/doc/tarpl/drop-flags.md

```diff
@@ -0,0 +1,81 @@
+% Drop Flags
+
+The examples in the previous section introduce an interesting problem for Rust.
+We have seen that's possible to conditionally initialize, deinitialize, and
+*reinitialize* locations of memory totally safely. For Copy types, this isn't
+particularly notable since they're just a random pile of bits. However types
+with destructors are a different story: Rust needs to know whether to call a
+destructor whenever a variable is assigned to, or a variable goes out of scope.
```

Comment:
Unless you have an owned variable, it's always unconditional though, right? Assuming l.v.a.l is just a subcase.